src/set.rs

```diff
@@ -1003,7 +1003,7 @@ where
     ///         match singles.entry(ch) {
     ///             Vacant(single_entry) => {
     ///                 // We found a new character for the first time.
-    ///                 single_entry.insert()
+    ///                 single_entry.insert();
```

Comment:
This exemplifies how it was a breaking change to return something other than `()`.